packages/hint/docs/contributor-guide/getting-started/events.md

```diff
@@ -3,7 +3,8 @@
 Information is shared internally via `event`s. `connector`s and `parser`s can
 create them, while `parser`s and `hint`s consume them.
 The following is a list of all the events common to all `connector`s, with
-their signature, and the `interface` they implement.
+their signature, and the `interface` they implement. The exception is the
+`local connector` that needs the `html parser` to emit these events:
```

Comment:
HTML parser

Reply:
You can use the newest GitHub feature for this changes 
![image](https://user-images.githubusercontent.com/606594/47334147-b51faf80-d63a-11e8-8219-1d2292148805.png) 😋

Reply:
Not on mobile.